commands.js

```diff
@@ -557,6 +558,7 @@ exports.commands = {
 			}
 		} else {
 			if (target === 'on' || target === 'true') {
+				if (room.modjoin) return this.errorReply("Modjoin is already turned on in this room.");
```

Comment:
How is that a different check though?

Also ya sure, I'll update this with that when I wake up.